RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java

```diff
@@ -67,14 +69,13 @@ public Store getStoreByID(@PathVariable int id, HttpServletResponse response) th
         }
     }
 
-   @PostMapping("/add")
-   @ResponseBody
-   @ResponseStatus(HttpStatus.CREATED)
-   public Store addStore(@RequestBody Store store) {
+  // @PostMapping("/add")
+  @RequestMapping(value = "/add", method = RequestMethod.POST, headers="Accept=application/json")
+  @ResponseStatus(value = HttpStatus.CREATED)
```

Comment:
no need to add value if you are not going to specify other properties of the annotation